### DISCUSSION TITLE
Hover over avatars to see names in poker

### DISCUSSION THREAD

**Discussion by drewhousman1010** [ParabolInc/parabol#6110] [Ideas]
A [user ](https://app.hubspot.com/contacts/3888472/contact/64282151) 🔒  writes:

> I can't see who has joined the meeting. All I see in the upper right corner are initials and when I hover over I can't see their names. I've asked the team to put up profile photos for now which helps.

I also can't see names on a hover during a poker meeting, but I can during a retro. 

I know we had this similar issue ([#4748](https://github.com/ParabolInc/parabol/issues/4748)) about showing names via hover during voting.

**Comment by tianrunhe** [comment]
I can't see names for avatar in retro either. This feels more like a bug than feature request (and it's different bug than #4748)
